src/index.html

```diff
@@ -4,9 +4,285 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin=" ">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <div class="moyo-container">
+      <header class="main-header">
+        <img
+          src="./images/logo.png"
+          alt="moyo-logo1"
+          class="moyo-logo1"
+        >
+        <nav class="main-nav">
+          <ul class="main-nav-list">
+            <li class="menu-list">
+              <a href="#" class="is-active" >Apple</a>
```

Comment:
Let's make links more informative
```suggestion
              <a href="#apple" class="is-active" >Apple</a>
```